packages/gateway/wrangler.toml

```diff
@@ -0,0 +1,39 @@
+# gateway.nft.storage wrangler config.
+name = "gateway-nft-storage"
+
+# `javascript` means our we'll send pre-built javascript code. 
+#  ...as opposed to `webpack` where wrangler builds our code for us.
+type = "javascript"
+
+account_id = ""
+watch_dir = "src"
+compatibility_date = "2021-12-03"
+
+[build]
+command = "npm run build"
+[build.upload]
+format = "service-worker"
+
+# PROD!
+[env.production]
+# name = "gateway-nft-storage-production"
+account_id = "fffa4b4363a7e5250af8357087263b3a" # Protocol Labs CF account
+zone_id = "fc6cb51dbc2d0b9a729eae6a302a49c9"    # nft.storage zone
+route = "*gateway.nft.storage/*"
+vars = { IPFS_GATEWAY = "https://ipfs.io" }
```

Comment:
Note: We will need to setup multiple gateways, either we need to hardcode multiple here or we will need to use a string array and parse it. Leaving for when needed